ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1594,7 +1617,11 @@ const RuntimePanel = fp.flow(
                   aria-label='Delete Environment'
                   disabled={disableControls || !runtimeExists}
                   onClick={() => setPanelContent(PanelContent.DeleteRuntime)}>Delete Environment</Link>
-              {runtimeExists || (pdExists && pdSizeReduced) ? renderNextButton() : renderCreateButton()}
+              {runtimeExists || (pdExists && pdSizeReduced)
+                  ? renderNextButton()
+                  : currentRuntime && currentRuntime.errors
```

Comment:
nit: could you use switchCase here instead? The double ternary hurts my eyes

Reply:
I like utils/index `cond()` in a case like this where it's a little more complex than a switch(constant)

Reply:
Ooh, `cond` is a good idea